### ISSUE TITLE
Accessibility keyboard navigation not allowing wrap around from first point to last point

### ISSUE THREAD

**Issue by XiLiuRoy** [highcharts/highcharts#16625]
#### Expected behaviour
When setting KeyboardNavigation->wrapAround to be true, and focus is on first data point, hit "left arrow" should move focus to the last data point of current series as per previous version, e.g. 9.1.2

#### Actual behaviour
When setting KeyboardNavigation->wrapAround to be true, and focus is on first data point, hit "left arrow" and focus stays on the first data point.

#### Live demo with steps to reproduce
https://jsfiddle.net/eo2vk4u5/7/ -> version 9.3.1

1. Hit "Tab" to focus on Pie chart
2. Hit "Left Arrow"
3. Focus is not moved

and comparing to previous version 9.1.2 https://jsfiddle.net/eo2vk4u5/4/
1. Hit "Tab" to focus on Pie chart
2. Hit "Left Arrow"
3. Focus moves to last data point of current series

#### Product version
 version 9.3.1

#### Affected browser(s)
Chrome, Firefox, IE

**Comment by KacperMadej** [comment]
Hi @XiLiuRoy ,

Thank you for reporting the problem.

**Comment by oysteinmoseng** [comment]
Fixed in pending PR #16732. The fix should be part of the next official release.

Closing this issue manually as our automation won't catch it when multiple issues are fixed in one PR.
